builders/tutorials/subsquid.md

```diff
@@ -0,0 +1,428 @@
+---
+title: Using Subsquid to Index Your ContainerChain
+description: Learn how to use Subsquid, a query node framework that can index both Substrate and EVM data, to process blockchain data for your Tanssi ContainerChain.
+---
+
+# Using Subsquid to Index Your ContainerChain
+
+## Introduction {: #introduction }
+
+[Subsquid](https://subsquid.io){target=\_blank} is a data network that allows rapid and cost-efficient retrieval of blockchain data from 100+ chains using Subsquid’s decentralized data lake and open-source SDK. In very simple terms, Subsquid can be thought of as an ETL (extract, transform, and load) tool with a [GraphQL](https://graphql.org/){target=\_blank} server included. It enables comprehensive filtering, pagination, and even full-text search capabilities.
```

Comment:
Sure I can fix. It is prettier that does this, so each time I run prettier it replaces all of these.

Reply:
Ah yeah, I've accidentally run prettier on a couple pages instead of the table formatter, and I know it messes up the tables for me. I don't know if you can configure it to use target=_blank instead, but even then, I feel like there may be other unintended side effects from using prettier